composer/datasets/cifar_hparams.py

```diff
@@ -183,25 +185,55 @@ class StreamingCIFAR10Hparams(DatasetHparams):
     """Streaming CIFAR10 hyperparameters.
 
     Args:
+        version (int): Which version of streaming to use. Default: ``2``.
         remote (str): Remote directory (S3 or local filesystem) where dataset is stored.
-            Default: ``'s3://mosaicml-internal-dataset-cifar10/mds/1/'``
+            Default: ``'s3://mosaicml-internal-dataset-cifar10/mds/2/'``
         local (str): Local filesystem directory where dataset is cached during operation.
             Default: ``'/tmp/mds-cache/mds-cifar10/'``
         split (str): The dataset split to use, either 'train' or 'val'. Default: ``'train'``.
     """
 
+    version: int = hp.optional('Version of streaming (1 or 2)', default=2)
```

Comment:
Can we update the following YAML's as well with an explicit version so it's explicit?
```
composer/yamls/models/resnet56_streaming_cifar10.yaml:  streaming_cifar10:
composer/yamls/models/resnet56_streaming_cifar10.yaml:  streaming_cifar10:
```

Reply:
Done